.github/workflows/push-trigger.yml

```
@@ -1,30 +1,63 @@
-      - '!release-branch'
       - master
       - 1.*
       - develop
       - MOSIP*
       - release*
       - ES-842
-
+pull_request:
+    types: [opened, reopened, synchronize]
+workflow_dispatch:
+  release:
```

actionlint (1.7.9)

could not parse as YAML: did not find expected key

(syntax-check)